tests/yakus/triplekan.test.cpp

```diff
@@ -0,0 +1,78 @@
+#include <gtest/gtest.h>
```

Comment:
Maybe also test with other types of melds as well.

Reply:
Added in 8ca3870ae0bf49165c9fc1220ded49941f8f0ccc